packages/web-runtime/src/store/user.js

```diff
@@ -98,8 +101,13 @@ const actions = {
           return
         }
 
-        const userGroups = await client.users.getUserGroups(login.id)
-        const user = await client.users.getUser(login.id)
+        const promiseUserGroups = client.users.getUserGroups(login.id)
```

Comment:
this should not happen in the background, we have some cases where it's needed to be in place before doing other things. as said before, we will touch the whole bootloader in the future and take care that all needed things happen in one place which then reduces the clutter

Reply:
sorry i re read my comment and your right, its confusing. Can you explain why we await after the function call?

Reply:
Imagine `doA` and `doB` call a network resource. Imagine that the resource of A takes 1ms and the resource of B 1.5ms.
If I do:
```
await doA()
await doB()
```
I have to wait for the full roundtrip of A to finish, before doing the roundtrip of B. This takes 1 + 1.5 ms
If I do:
```
a = doA()
b = doB()
await a
await b
```
The roundtrip of A and B happen at ~ the same time. So it takes me 1.5ms to do both.
If I call these functions inside `doAll`, the external behavior is the same: `doAll` finishes only after `doA` and `doB` have finished, but it takes 40% less time to run it.